src/components/Dialogs/Dialogs.js

```diff
@@ -12,8 +12,20 @@ const Dialogs = (props) => {
   const { dialogs, messages } = state;
 
 
-  const dialogsElements = dialogs.map((dialog) => <DialogItem name={dialog.name} id={dialog.id} />);
-  const messagesElements = messages.map((message) => <Message message={message.message} />);
+  const dialogsElements = dialogs.map((dialog) => (
+    <DialogItem
+      name={dialog.name}
+      key={dialog.id}
+      id={dialog.id}
+    />
+
+  ));
+  const messagesElements = messages.map((message) => (
+    <Message
+      key={message + Math.random(1000)}
```

Comment:
It seems that "message" here is an object, be careful, remember the case of "[object Object]".

Reply:
it is really necessary